python/lsst/daf/base/propertyContainer/propertyContainerContinued.py

```diff
@@ -62,28 +62,17 @@ def _propertyContainerGet(container, name, returnStyle):
     name : ``str``
         Name of item
     returnStyle : ``ReturnStyle``
-        Control whether data is returned as an array or scalar:
+        Control whether numeric or string data is returned as an array
```

Comment:
I got myself confused when reconciling the docs and commit message. I understand:
(1) `getArray(nameofAPropertyList)` should return a scalar that one can iterate over. 
(2) If `ReturnStyle.SCALAR`, then it returns the last value? 

Does this mean the scalar reference in #1 is a different concept than #2? Because I would naively think: the property list is being returned as a scalar, which means I get the last value.

Reply:
The point of iterating (in my commit message) is that if you want to iterate over all items in a `PropertySet` or `PropertyList` then `getArray` is the approved way to do it because it returns all numeric or string data (instead of ignoring all but the last value, like `getScalar`. However, before this commit `getArray` would raise an exception for items that were neither numeric nor strings, which meant iterating over all items was a very messy business (you had to determine the data type first, then call getArray if numeric or string). Now you can just call `getArray` on *any* item and get something sensible back -- something that contains all known information about that item.